dev-libs/benchmark/benchmark-1.3.0.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="A microbenchmark support library"
+DESCRIPTION="A library to support the benchmarking of functions, similar to unit-tests."
+HOMEPAGE="https://github.com/google/benchmark/"
+COPYRIGHT="2015-2017 Google Benchmark developers"
+LICENSE="Apache v2"
+REVISION="1"
+SOURCE_URI="https://github.com/google/benchmark/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="f19559475a592cbd5ac48b61f6b9cedf87f0b6775d1443de54cfe8f53940b28d"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	benchmark$secondaryArchSuffix = $portVersion
+	lib:libbenchmark$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	benchmark${secondaryArchSuffix}_devel = $portVersion
+	devel:libbenchmark$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	benchmark$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	mkdir -p build && cd build
+
+	cmake $cmakeDirArgs \
+		-DCMAKE_BUILD_TYPE='Release' \
+		-DBUILD_SHARED_LIBS:BOOL=ON \
+		..
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build
+	make install
+
+	mkdir -p $includeDir
+	mv $prefix/include/* $includeDir/
+	rm -rf $prefix/include/
+
+	prepareInstalledDevelLibs libbenchmark
```

Comment:
Ok, I've obviously done something incorrect here.  I'm going to remove this pull request and try again.